src/core/auth.ts

```diff
@@ -1,7 +1,6 @@
+import type { AxiosRequestConfig, AxiosResponse } from 'axios'
```

Comment:
Even if current request adaptor is axios, isn't it better to keep abstraction from types? (alias HTTPRequest to AxiosRequestConfig, etc)

Reply:
Yeah, I agree!

Reply:
Refactored in commit 1171a8ac